app/models/users.py

```diff
@@ -0,0 +1,71 @@
+import tinydb
+from werkzeug.security import generate_password_hash
+from flask_login import UserMixin
+from app import util
+from app.models.db import users
+
+
+class ErrNotFound(Exception):
+    pass
+
+
+class ErrInvalidParameters(Exception):
+    pass
+
+
+class User(UserMixin):
+    def __init__(self, id, username):
+        self.id = id
+        self.username = username
+
+    def get_id(self):
+        # This function is requried by Flask-Login
+        return self.id
+
+
+def insert(username="", password=""):
+    user_id = util.random_id(initial="U")
```

Comment:
nit; maybe we can user smaller letter 'u' instead of the capital U

Reply:
This has fixed